### ISSUE TITLE
About Us Page

### ISSUE THREAD

**Issue by alexandrastubbs** [hackforla/website#614]
### Overview
Create an 'About Us' page to host information on the organization, primarily built for partner/sponsor audience, but also for individuals or potential volunteers looking to learn more about what we do. 

### Action Items

Information Needed
- [x] Bonnie to share summary blurb for the page
- [x] Bonnie to share summary on workforce development
- [x] Bonnie to share list of partners
- [ ] Bonnie will write letter

Design
- [x] Low fidelity mock-up 
- [x] Review with product 
- [x] High fidelity mock-up
- [x] Review with product 

Development
- [x] Create page based on Figma mockup
- [ ] Add to menu of HfLA site - desktop and mobile
- [ ] Before publish, ask Bonnie for letter.

### Resources/Instructions

Dashboard page: https://hackforla.org/dashboard

**Comment by ExperimentsInHonesty** [comment]
Partners (past and current):
Yale OpenLab, https://openlab.yale.edu/, 
Empower LA
Digital Defense Fund
Point Source Youth 
SPY
LADOT
City of Santa Monica
NDICA
EnCiv.org
Balotopedia
LA City Attorney's office
LA Mayors Office
LA City Department of Personnel
LA County Homeless Initiative
LA Metro
Hope of the Valley

**Comment by alexandrastubbs** [comment]
Add in a projects section and include some cards (tiny) similar to Hack Nights page.

**Comment by leestella748** [comment]
@ExperimentsInHonesty @alexandrastubbs 

I added a projects section and added a few partner logos just to see what it would look like when populated with the actual logos.  

Figma: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=2003%3A817

**Comment by alexandrastubbs** [comment]
- Add icons to accomplishments

**Comment by leestella748** [comment]
Icons added!  Let me know if this is what you meant and if there are any other changes you'd like to see 
<img width="218" alt="image" src="https://user-images.githubusercontent.com/61727650/89359457-bd63a600-d67a-11ea-82a6-71ebc8c74485.png">

**Comment by alexandrastubbs** [comment]
We will be adding the entire want to know more about HfLA section from the current homepage to the bottom of this page. We will also be adding a CTA to the footer that drives here.

**Comment by alexandrastubbs** [comment]
@leestella748 

Bonnie & I took a look at the Figma. Can we make the 'want to know more' section more like the section on the current website with card on card? Screenshot pasted here. 

![Screen Shot 2020-08-12 at 5 36 21 PM](https://user-images.githubusercontent.com/66918724/90070762-62f1c780-dcc2-11ea-933c-b45a3da9cd71.png)

**Comment by alexandrastubbs** [comment]
Some other notes -- where we currently have workforce development, we would like to replace with 'Hack for LA Pillars' and then include the following information below (possibly in an accordion format given the quantity of information): 

Workforce Development: Enlightened self-interest is a powerful motivator.  Our members have access to mentorship, one-on-one help with job searches, and resume coaching.  We do project matching to fit skill and subject areas that they wish to develop new skills, gain confidence in newly learned skills, transition from beginner to junior, to mid-level, or expert.  And often in areas they wish to add impressive credits to their resume.
Impact: We choose and empower projects where there is a demonstrable impact on stakeholders, project partners, our local community and our members.
Learning: an environment where every member and the org itself benefit from the shared experiences of the members.  From high school students to Ph.D., from novice to expert, there is knowledge to be gained by sharing practices and working collaboratively, making mistakes and learning from them in an environment that values failing fast and learning.
Diversity Equity and Inclusion: The diverse experiences of all of our members help us create a better civic tech ecosystem, by bringing their lived experience to our org.  Our diversity includes race, sexual orientation, learning modalities, cognitive abilities, and socio-economic experiences.  Our model of inclusion requires and inspires empathy and patience, both with others and ourselves.  We are all here to learn to work together to create the change we want to see in the world, and that happens one conversation at a time.
Giving: We all come together to give freely of our time, expertise and interest and empathy, in an effort to improve our local community through direct action.
Agility: There will be impediments, setbacks and disappointments. We embrace change by empowering our members to meet, talk, experiment and achieve enough common consensus to move internal initiatives forward, without rigid structures and oppressive processes.  This works on project teams, Communities of Practice and Organizational Stewardship.
Sustainability: to create a sustainable community that provides consistent results and experience for stakeholders and a feeling of stability amongst its members.
Scalability: In order to scale our impact with people and projects, we experiment, iterate and refine, measure and automate. 

@ExperimentsInHonesty will follow-up with the copy for Accomplishments & Metrics.

**Comment by alexandrastubbs** [comment]
Last note - we need to find a place on this page to say "Hack for LA support’s LA’s Sustainable Development Goals (SDG) movement" and link to https://sdg.lamayor.org/. 

We should include the logo as well: https://sdg.lamayor.org/sites/g/files/wph1131/f/styles/medium/public/LA.png?itok=NGcr9CbL

**Comment by alexandrastubbs** [comment]
During our stand up, we discussed the following: 

**@daniellen00 :** 
- turning the top portion of the page on the right where we have the 'Hack for LA Platform' information into a letter format
- Try a version of the page with a menu at the top 

**@ExperimentsInHonesty :**
- gather all the partner logos and place in this folder: https://drive.google.com/drive/u/1/folders/1DBgFWGwreCKaZoh29Xe7OG92swy-uE1L
- update accomplishments (specifically the national day of civic hacking) and adding links to each of the project pages

**Needs to be assigned to developer:** 
- Make sure that you put the trophy image from the wins section on the credits page (https://thenounproject.com/search/?q=trophy&i=1527058)

**Comment by daniellen00** [comment]
Progress: Page is almost ready- pending decisions on Table of Contents / HfLA Platform layouts (plus Letter from CEO)

**Comment by daniellen00** [comment]
Next steps: 
- HfLA Platform decision: columns with icons
- Header: Get rid of dark background
- Table of contents: Remove bulletpoints, create more side/top final options to choose from
- @ExperimentsInHonesty will write Letter from CEO

**Comment by ExperimentsInHonesty** [comment]
need mobile

**Comment by alexandrastubbs** [comment]
@daniellen00 

Notes on mobile (we know it is still a WIP): 

1. For letter, it will be too long to show full letter on mobile, so maybe we use an accordion and just have 'letter from the editor' and then show something along the lines of 'Thank you for coming to the site. This letter is fairly long on mobile. If you'd like to read the whole thing, click here.' Upon clicking, the visitor can read the full letter. Bonnie will explain more during Sunday meeting.
2. The page will be too long on mobile with all of the content laid out as it is on web. We should employ something similar to what we have on Hack Nights where everything is initially collapsed and you can view the sub-categories from a first glance.

**Comment by alexandrastubbs** [comment]
@daniellen00 do you have an update on this?

**Comment by daniellen00** [comment]
@alexandrastubbs Hi Alex, I'm really sorry about the delay, thank you for checking in- it's been a little crazy catching up since returning from my camping trip- thank you for the notes above, I completely agree. I am 95% finished with the full expanded mobile page, and after that I will finish the collapsed version right next to it, which should be really simple. I should be done early tomorrow (Sat) morning (tonight I just have Rosh hoshana). I'll post an update here as soon as I finish tomorrow morning.

**Comment by daniellen00** [comment]
@alexandrastubbs @ExperimentsInHonesty I am almost finished but I just wanted to get your thoughts ([link](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1803%3A13692)) - in the collapsed version do you want every section collapsed, or should I keep sections like the quote and links always expanded (as currently shown in the Figma)?

**Comment by ExperimentsInHonesty** [comment]
@ExperimentsInHonesty will provide preview text for letter for mobile version

**Comment by ExperimentsInHonesty** [comment]
- changing the read about us to be all on one card
- changing icons on mobile menu

1.  - platform
2. - sponsors
3. - usability hub test for donating

**Comment by alexandrastubbs** [comment]
We are moving the Join Us information to a new page. I will make an issue for this.

**Comment by ExperimentsInHonesty** [comment]
I have signed off on design and this page can go to development

**Comment by drubgrubby** [comment]
@alexandrastubbs @ExperimentsInHonesty - The header text for the about us page and the getting started page is almost identical, yet not quite identical.  Is that intentional?  Do we want them to be more different? More the same? 

I got the text for this off of the figma.  (I'm just getting started, so please don't judge anything else yet ;-)

<img width="1440" alt="Screen Shot 2020-10-14 at 7 54 06 PM" src="https://user-images.githubusercontent.com/68244054/96071618-60912280-0e57-11eb-8f18-ff4333a5e3a9.png">
<img width="1440" alt="Screen Shot 2020-10-14 at 7 54 20 PM" src="https://user-images.githubusercontent.com/68244054/96071625-638c1300-0e57-11eb-977a-2cd117b495a7.png">

**Comment by daniellex0** [comment]
@drubgrubby Hi David, sorry I know this isn't directed at me but I got an alert about this comment- the header title seems to be the same (48px bold) but do you mean the left-aligned header paragraph and text size? Or the layout with the image to the right? If you mean those, yes this page's header layout is meant to look slightly different from the Getting Started page one so it should be fine, but thank you for double checking! if you meant something else though, let me know!

**Comment by drubgrubby** [comment]
@daniellex0 - Thanks for responding.   I'm actually talking about the content of the text that's under the title. The About us page is, "We bring together civic-minded volunteers to build digital products...", and the Getting Started page says, "Hack for LA brings together civic-minded volunteers who build digital products...".  It seems strange to me that the text on these two pages is so close, but not identical.  It seems they should either be more the same or more different. 

But thanks for your answers to questions that I would have gotten around to asking at some point and now I don't have to.

**Comment by alexandrastubbs** [comment]
@drubgrubby the text is intentional! There will ultimately be a rewrite of the getting started page later so we can make them consistent then. Thanks for flagging!

**Comment by drubgrubby** [comment]
Progress:
Getting up and running on this.  The header is done and I have a "lorem ipsum" page up with all of the sections.  I'm working on the sticky navigation and making each section open and close for mobile, and hope to have at least that done this week.  After that I will begin to populate the sections with the appropriate information, cursing as I go, no doubt. 

Blockers:
Other than the things I don't yet know and the skills I don't yet have, and the annoying need for sleep, there are no blockers.  

ETA
I do not yet have a reliable ETA; it lies shrouded in the mist of the future. My guess is two weeks, which probably means three.

**Comment by drubgrubby** [comment]
**Progress**
The JavaScript for the sticky side nav-bar is (mostly) done.  I hit some roadblocks along the way and ended up re-doing the content part of the page in grid, rather than flexbox, so it took longer than I hoped. 

Next up: Doing the mobile coding so the content appears and disappears as per the figma.  After that I'll fill in the content for the various section. 

**Blockers**
Still working on the coding to make time pass more slowly, or perhaps to have more hours in the day. That, and that PG&E is likely to shut off our power for two days this week. 

**ETA**
Let's say two more weeks.

**Comment by drubgrubby** [comment]
**Progress**
The JavaScript programming for nav, accordion, and misc other things is done.  

I have begun adding content to the various sections...well, more accurately, I've added content to one section.  The beatings will continue until morale improves. (I'll keep adding content until all of the content has been added). 

**Blockers**
Overwhelming pre-election anxiety. 

**ETA**
Hopefully I'll have a PR by next week.

**Comment by drubgrubby** [comment]
**Progress**
To paraphrase Kirsten Dunst in Elizabethtown, "I failed.  I failed, I failed, I failed, I failed, I failed.  I failed."  

For a variety of reasons, both internal and external, I did not get this page completed this week.  Rather than spend the coming week wearing a hair-shirt and enumerating my failings, I will instead redouble my efforts to complete this issue.

**Blockers**
I love the design of this page and think it's going to be really beautiful.  However, the fact that each section of this design has a different layout makes doing each section only slightly less work than doing a whole page that is a single design.  That's not in any way a criticism - as I said, I love the design of this page - but just something to be aware of when designing pages.

**ETA**
I am recommitting to my intention of finishing this page by next week.  (Wish me luck).

**Comment by drubgrubby** [comment]
@alexandrastubbs @daniellex0 - I just want to confirm that this text from the figma is the correct text for "National Day of Civic Hacking" ;-)  If not, where might I find actual text to use? 

<img width="358" alt="Screen Shot 2020-11-09 at 6 22 41 PM" src="https://user-images.githubusercontent.com/68244054/98619876-f6826680-22b8-11eb-8d80-bcca0b7cb2df.png">

**Comment by ExperimentsInHonesty** [comment]
@drubgrubby lol

**Comment by alexandrastubbs** [comment]
Definitely not the right text lol... I did not create copy for this page, @daniellen00 any thoughts here?

**Comment by daniellex0** [comment]
@drubgrubby haha yikes, I think that was what the original copy said and I forgot to point it out, good catch! @ExperimentsInHonesty @alexandrastubbs Should I move it under the 2020 column and include some of [these](https://www.codeforamerica.org/events/national-day-of-civic-hacking-2020) details from this year's event? 

Something like:
"In 9/2020 Hack for LA joined over 40 Code for America brigades in bringing together civic leaders, public servants, designers, coders, and engaged citizens to partner with local government and community groups to help those most in need of a social safety net services during COVID-19."

For context here are all the other blurbs:
<img width="590" alt="Screen Shot 2020-11-10 at 10 02 31 AM" src="https://user-images.githubusercontent.com/67400366/98713020-e8c1f500-233b-11eb-94c0-09e495f51045.png">

**Comment by ExperimentsInHonesty** [comment]
Sorry all.  I created the copy.  It's just a placeholder.  We really should make it so this section gets its text from a markdown file so I can change it out as needed @drubgrubby @daniellen00 @alexandrastubbs

**Comment by drubgrubby** [comment]
@ExperimentsInHonesty - It would be great to pull the content for this section from a markdown file of some sort.  I'm going to suggest that we make that a later version of the page and a separate issue.

**Comment by drubgrubby** [comment]
@alexandrastubbs @daniellex0 @ExperimentsInHonesty
Where do the links in this point to?
-> Survey results
-> See app
-> Learn more  

<img width="541" alt="Screen Shot 2020-11-11 at 5 36 16 PM" src="https://user-images.githubusercontent.com/68244054/98884278-7ee14280-2444-11eb-802d-9595b930cd7e.png">

**Comment by ExperimentsInHonesty** [comment]
There used to be info in the figma file about where the links go.  Can you check to see if it's missing now.  If so I'll redocument, but would prefer not to.  Sent from my phone @drubgrubby

**Comment by daniellex0** [comment]
Sorry just saw this- @drubgrubby I just copied and pasted the links right next to the final figma mockup for your reference

**Comment by drubgrubby** [comment]
**Progress**
The About Us page is ABD (All but done).  All of the content has been added to the various sections, there is programming in place for the various things that need programming...with the exception of: 
1) The little arrow on the sticky navigation isn't where it's supposed to be (hopefully the dev team can help me sort that out today). 
2) The accordion javascript needs to be amended for the special circumstance of the "Letter..."

**Punch list**
These are the things that I can think of that are currently standing between now and a Pull Request: 
- Content for the Letter from the Executive Director
- Text for the "National day of civic hacking" under the accomplishments section
- Sticky nav arrow alignment
- Letter from editor JavaScript adjustment
- 4000 bugs and other design things that I don't know about yet

**The code-changing magic of tidying up**
In the push to get this "done" I created more than a few unnecessary classes and wrote some ugly javaScript.  This doesn't need to be cleaned up for the page to work, but I'm uncomfortable with the future cursing of my name that will come from the next person to work on this page if I don't do a bit of cleanup. 

**ETA**
One week plus the amount of time it takes to get the text for the Letter from the Executive Director ;-)

**Comment by drubgrubby** [comment]
**Progress**
So close...

I put in a PR.  There are still a couple of things to complete before code review, and merging.  

https://github.com/hackforla/website/pull/811#issue-525327993

**Blockers**
The linear nature of time. 

**ETA**
Wednesday this should be ready for code review and follow-up visual review.

**Comment by ExperimentsInHonesty** [comment]
@drubgrubby - confirming that you saw the comment on the PR https://github.com/hackforla/website/pull/811#issuecomment-733452961

**Comment by drubgrubby** [comment]
**Progress:** 
I have completed the corrections from Danielle's visual review.

I have to re-do the script for the accordion on the letter from the executive director as that section is different and I don't seem to be able to shoehorn the current code into it. 

I cannot figure out how to make the about-us.js file work from the assets folder.  It is currently in the _includes folder.  Need some help with this. 

**Still need:** 

- Text for letter from Executive Director
- Text for National Day of Hacking on the Accomplishments section
- Animated gif for the donations section (copying it from figma doesn't seem to work)

**Questions**
Are the titles in the "Accomplishments" section supposed to be hyperlinks?

**ETA**
I'll have the javascript done by Tuesday.  Then waiting on things.

**Comment by ExperimentsInHonesty** [comment]
@drubgrubby yes the project titles on the accomplishment section should be links to the project pages

**Comment by drubgrubby** [comment]
@cnk - https://github.com/drubgrubby/website/tree/dar_about-us-page 
I'm trying to figure out how to move the _includes/about-us.js into the assets folder, but can't make it work.  All of the things I've tried are at the bottom of about-us.html.

**Comment by ExperimentsInHonesty** [comment]
@daniellen00 to add animated gif to google drive and put the link here

**Comment by drubgrubby** [comment]
@ExperimentsInHonesty - I'm adding hyperlinks to the accomplishments page.  For most of them I linked to the project page on hackforla.org.  There were two that I wasn't sure where to point the link.  This is what I did, let me know if you would like to link to anything else. 

National Day of Civic Hacking: https://www.codeforamerica.org/events/national-day-of-civic-hacking-2020
Climate Collabathon: https://www.collabathon.openclimate.earth/

**Comment by drubgrubby** [comment]
I'm done with everything I can do (or everything I'm going to do) until I hear from reviewers and get the couple of things I'm waiting on. 

Waiting on: 
- Text of Letter from the Executive Director
- Animated gif for the donation section

Thank you for your support.
